doc/log-processing.md

```diff
@@ -0,0 +1,101 @@
+# Processing and Analyzing Conversation Logs
+
+This document details the procedures to analyze the commands
+collected from users who have agreed to participate in our
+research and share their commands with us.
+
+There are two type of logs:
+- "passive" logs, which contain only the delexicalized context
+  + command. These logs are enabled by agreeing to the consent form
```

Comment:
Is "+" a typo?

Reply:
No it's "context + command", just wrapped weird. Let me fix that...